app/controllers/api/v1/authentication_controller.rb

```diff
@@ -190,6 +189,24 @@ def auth_params
       params.require(:user_auth).permit(attributes)
     end
 
+    def render_invalid_mobile
+      attr = I18n.t('activerecord.attributes.user.mobile')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.